vpd_tool_impl.cpp

```diff
@@ -942,99 +1025,6 @@ void VpdTool::parseSVPDOptions(const nlohmann::json& json)
-                        }
-                        else
-                        {
-                            printFixSystemVPDOption(
-                                VpdTool::BMC_DATA_FOR_CURRENT);
-                            printFixSystemVPDOption(
-                                VpdTool::SYSTEM_BACKPLANE_DATA_FOR_CURRENT);
-                            printFixSystemVPDOption(VpdTool::NEW_VALUE_ON_BOTH);
-                            printFixSystemVPDOption(VpdTool::SKIP_CURRENT);
-                            printFixSystemVPDOption(VpdTool::EXIT);
-                        }
-
-                        cin >> option;
-                        cout << '\n' << outline << endl;
-
-                        EditorImpl edit(constants::systemVpdFilePath, json,
-                                        get<1>(data), get<2>(data));
-                        string value;
-
-                        switch (option)
-                        {
-                            case VpdTool::BMC_DATA_FOR_CURRENT:
-                                edit.updateKeyword(toBinary(get<3>(data)), 0,
-                                                   true);
-                                cout << "\nData updated successfully.\n";
-                                break;
-                            case VpdTool::SYSTEM_BACKPLANE_DATA_FOR_CURRENT:
-                                edit.updateKeyword(toBinary(get<4>(data)), 0,
-                                                   true);
-                                cout << "\nData updated successfully.\n";
-                                break;
-                            case VpdTool::NEW_VALUE_ON_BOTH:
-                                cout << "\nEnter the new value to update both "
-                                        "on "
-                                        "BMC & "
-                                        "System Backplane (Value should be in "
-                                        "ASCII or "
-                                        "in HEX(prefixed with 0x)) : ";
-                                cin >> value;
-                                cout << '\n' << outline << endl;
-
-                                edit.updateKeyword(toBinary(value), 0, true);
-                                cout << "\nData updated successfully.\n";
-                                break;
-                            case VpdTool::SKIP_CURRENT:
-                                cout
-                                    << "\nSkipped the above record-keyword "
-                                       "pair. "
-                                       "Continue to the next available pair.\n";
-                                break;
-                            case VpdTool::EXIT:
-                                cout << "\nExit successfully\n";
-                                exit(0);
-                            default:
-                                cout << "\nProvide a valid option. Retrying "
```

Comment:
don't we need a break in default case

Reply:
I discussed with Priyanga, it seems to be not intended to have break here.